tests/phpunit/tests/xmlrpc/wp/newPost.php

```diff
@@ -284,11 +330,11 @@ function test_terms_names() {
 		$this->assertContains( $cat2->term_id, $post_cats );
 
 		// create a second post attempting to use the ambiguous name
-		$post2 = array(
-			'post_title' => 'Test',
+		$post2   = array(
+			'post_title'  => 'Test',
 			'terms_names' => array(
-				'category' => array( $cat1_name, $ambiguous_name )
-			)
+				'category' => array( $cat1_name, $ambiguous_name ),
```

Comment:
Should items in this array start on new lines since it has more than one item?

Reply:
It's not an associative array.